roles/edpm_neutron_dhcp/templates/wrappers/haproxy.j2

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+{% if edpm_neutron_dhcp_sidecar_debug -%}
+set -x
+{%- endif -%}
+
+ARGS="$@"
+
+IMAGE_NAME={{ edpm_neutron_dhcp_sidecar_haproxy_image_name }}
+NEUTRON_CONFIG_DIR={{ edpm_neutron_dhcp_agent_config_dir }}
+NEUTRON_LIB_DIR={{ edpm_neutron_dhcp_agent_lib_dir }}
+
+{% raw -%}
+# Extract the network namespace UUID from the command line args provided by
```

Comment:
The logic of wrappers is largely repetitive. I wonder if we could consolidate it somehow.